# GetLitterList.srv
---
bumperbot_detection/LitterList remembered_litter  